tools/vf-component-library/src/site/troubleshooting/index.yml

```diff
@@ -0,0 +1,3 @@
+common:
```

Comment:
Does this file do anything? Accidental commit?

Reply:
hmmm, pass

Reply:
I'm not sure if this is me - I'm not sure if it's doing anything -- remove the whole lot?

Reply:
+1

Reply:
removed